gdal/swig/python/osgeo/utils/gdal_calc.py

```diff
@@ -620,7 +622,7 @@ def main(argv):
     parser.add_option("--quiet", dest="quiet", action="store_true", help="suppress progress messages")
     parser.add_option("--optfile", dest="optfile", metavar="optfile", help="Read the named file and substitute the contents into the command line options list.")
 
-    # parser.add_option("--color_table", dest="color_table", help="color table file name")
+    parser.add_option("--color_table", dest="color_table", help="color table file name")
```

Comment:
For consistence with --creation-option:

```suggestion
    parser.add_option("--color-table", dest="color_table", help="color table file name")
```